actions/snap-upgrade.py

```diff
@@ -160,6 +162,11 @@ def has_migrated_from_deb():
     backup = action_get('backup')
     channel = config('channel')
 
+    # We need to determine a default fallback channel since, there isn't an "auto" channel
+    channel = get_target_etcd_channel()
```

Comment:
You can remove this line right?

```suggestion
    # We need to determine a default fallback channel since, there isn't an "auto" channel
    channel = get_target_etcd_channel()
```

Reply:
`channel` gets passed to the snap command as the refresh channel. The charm has an "auto" setting for the channel, and the appropriate channel is determined by `get_target_etcd_channel()`. Not having this breaks the snap command since there isn't an `auto` channel for etcd.

Reply:
no, sorry -- i noticed on line `163` there's an assignment to `channel`, then line 166 there's another assignment to `channel`.  it seems like the first assignment doesn't do anything

Reply:
Oh yeah, I just now realized that. Thanks, will update that asap.